src/cheri/insns/sh123adduw_32bit.adoc

```diff
@@ -34,11 +34,13 @@ Encoding::
 ....
 
 {cheri_cap_mode_name} Description::
-Increment the address field of `cs2` by the unsigned word in `rs1` shifted left
-by _n_ bit positions and write the result to `cd`. The valid tag bit of the output
-capability is 0 if `cs2` did not have its tag set to 1, the incremented address
-is outside `cs2` 's <<section_cap_representable_check>> or `cs2` is sealed.
-
+Increment the address field of the capability `cs2` by the unsigned word in `rs1`
```

Comment:
We aren't actually modifying cs2 here do we need to add the copy step like the sentry instruction?

Reply:
yep - and the other similar instructions also needed fixing